conda/napari_cellseg3d_ARM64.yml

```diff
@@ -11,18 +11,14 @@ dependencies:
     - pip:
         - numpy
         - napari>=0.4.14
-        # - opencv-python>=4.5.5
         - scikit-image>=0.19.2
         - matplotlib>=3.4.1
         - tifffile>=2022.2.9
-        # - imageio-ffmpeg>=0.4.5
         - torch>=1.11
         - monai[nibabel, einops]>=0.9.0
         - tqdm
-        # - nibabel
         - scikit-image
-        # - pillow
         - pyclesperanto-prototype
         - tqdm
         - matplotlib
-        # - vispy>=0.9.6
+        - napari_cellseg3d
```

Comment:
The problem with this is that no Qt backend (PySide/PyQt) is installed. If people try to launch directly, it will not work. I will update install instructions accordingly

Reply:
can add then to the call, no? `- napari_cellseg3d[pyqt5]`